src/com/seafile/seadroid2/ui/fragment/StarredFragment.java

```diff
@@ -133,7 +133,13 @@ public void refreshView() {
             mPullRefreshListView.onRefreshComplete();
             Toast.makeText(mActivity, getString(R.string.network_down), Toast.LENGTH_SHORT).show();
         }
-        ConcurrentAsyncTask.execute(new LoadStarredFilesTask(getDataManager()));
+        List<SeafStarredFile> starredFiles = getDataManager().getCachedStarredFiles();
+        boolean refreshTimeout = getDataManager().isStarredFilesRefreshTimeout();
+        if (mRefreshType==REFRESH_ON_PULL || starredFiles == null || refreshTimeout)  {
```

Comment:
the refresh button on action bar does\`t work any more, change if condition like this
```
if (mRefreshType == REFRESH_ON_PULL
                || mRefreshType == REFRESH_ON_OVERFLOW_MENU
                || starredFiles == null
                || refreshTimeout)  {
```